blocks/components/editable/index.js

```diff
@@ -1,3 +1,98 @@
-export default function Editable( { value } ) {
-	return wp.element.createElement( 'p', null, value );
+const el = wp.element.createElement;
+let editorInstance = 1;
+
+export default class Editable extends wp.element.Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.onFocusIn = this.onFocusIn.bind( this );
+		this.onFocusOut = this.onFocusOut.bind( this );
+		this.id = `tinymce-instance-${ editorInstance }`;
+		editorInstance++;
+	}
+
+	componentDidMount() {
+		this.initialize();
+		if ( this.props.focusConfig ) {
+			this.focus();
+		}
+	}
+
+	initialize() {
+		const config = {
+			mode: 'exact',
+			elements: this.id,
+			theme: false,
+			inline: true,
+			toolbar: false,
+			entity_encoding: 'raw',
+			setup: this.onSetup,
+			formats: {
+				strikethrough: { inline: 'del' }
+			}
+		};
+
+		tinymce.init( config );
+	}
+
+	onSetup( editor ) {
+		this.editor = editor;
+		editor.on( 'init', this.onInit );
+		editor.on( 'focusIn', this.onFocusIn );
+		editor.on( 'focusout', this.onFocusOut );
+	}
+
+	onInit() {
+		this.editor.setContent( this.props.value );
+	}
+
+	onChange() {
+		const value = this.editor.getContent();
+		if ( value === this.props.value ) {
```

Comment:
Using `isDirty` coupled with `save` is possible I think 👍
And yeah most calls will pass the condition here, I think